next-cloudinary/package.json

```diff
@@ -2,13 +2,30 @@
   "name": "next-cloudinary",
   "version": "6.13.0",
   "license": "MIT",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
-  "source": "src/index.ts",
+  "main": "dist/next-cloudinary.cjs.js",
```

Comment:
are there any implications of not using index as the name?

Reply:
I can't imagine any implications of that. Index is just a filename like any other in this context.